### PR TITLE
FLEEK-243 Adds Ocata Cell Generation and Nova/Cinder migrations

### PR DESCRIPTION
**Pull request by antonym**
Adds Cells v2 DB
Adds initial Cells v2 cells
Does Cinder/Nova migration from Ocata venv
Removes deletion of openstack-ansible wrapper in Ocata prepare as
it should be overwritten during the next bootstap

### CODE REVIEW

File: incremental/playbooks/create-cell0.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        login_user: "{{ galera_root_user }}"
+        login_password: "{{ galera_root_password }}"
+        login_host: "{{ nova_galera_address }}"
+        name: "{{ item['name'] }}"
+        password: "{{ item['password'] }}"
+        host: "{{ item['host'] }}"
+        state: "present"
+        priv: "{{ item['database'] }}.*:ALL"
+        append_privs: "{{ item['db_append_privs'] | default(omit) }}"
+      with_items:
```
Comment (rexredinger) [inline incremental/playbooks/create-cell0.yml]: Could the with_items clause be simplified to only list the host entries?

Comment (antonym) [reply incremental/playbooks/create-cell0.yml]: Yeah, it probably could.  I copied some of this from upstream and it was set to do additional items as well so there's probably a few things we could compress.  It worked though so I haven't tried optimizing it yet.

**Comment by antonym** [comment]
recheck_xenial_mnaio-snap_swift_pike_to_queens_inc

**Comment by antonym** [comment]
recheck_xenial_mnaio-snap_swift_pike_to_queens_inc

**Comment by antonym** [comment]
Merging this in... I've added N2P tests in addtion to P2Q that will now gate on PRs so we'll get some more exercising on this with a few more PRs I'm putting up.

**Pull request merged**